### DISCUSSION TITLE
Is there any way to obtain the startup parameters for llama-server.exe

### DISCUSSION THREAD

**Discussion by neavo** [ggml-org/llama.cpp#9053] [General]
Is there any way to obtain the startup parameters for llama-server.exe, such as the value of np?
This way, downstream applications that use this interface can automatically optimize parameters without requiring users to manually modify them.

**Comment by ggerganov** [comment]
Currently no, but we could expose those through an endpoint.

**Reply by neavo** [reply]
> Currently no, but we could expose those through an endpoint.

wait for this
